oneview_redfish_toolkit/tests/blueprints/test_session.py

```diff
@@ -0,0 +1,198 @@
+        util.load_config('redfish.conf')
+
+        # creates a test client
+        self.app = Flask(__name__)
+
+        self.app.register_blueprint(session_blueprint)
+
+        @self.app.errorhandler(status.HTTP_500_INTERNAL_SERVER_ERROR)
+        def internal_server_error(error):
+            """General InternalServerError handler for the app"""
+
+            redfish_error = RedfishError(
+                "InternalError",
+                "The request failed due to an internal service error.  "
+                "The service is still operational.")
+            redfish_error.add_extended_info("InternalError")
+            error_str = redfish_error.serialize()
+            return Response(
+                response=error_str,
+                status=status.HTTP_500_INTERNAL_SERVER_ERROR,
+                mimetype="application/json")
+
+        @self.app.errorhandler(status.HTTP_400_BAD_REQUEST)
+        def bad_request(error):
+            """Creates a Bad Request Error response"""
+            redfish_error = RedfishError(
+                "PropertyValueNotInList", error.description)
+
+            redfish_error.add_extended_info(
+                message_id="PropertyValueNotInList",
+                message_args=["VALUE", "PROPERTY"],
+                related_properties=["PROPERTY"])
+
+            error_str = redfish_error.serialize()
+            return Response(
+                response=error_str,
+                status=status.HTTP_400_BAD_REQUEST,
+                mimetype='application/json')
+
+        self.app = self.app.test_client()
+
+        # propagate the exceptions to the test client
+        self.app.testing = True
+
+    @mock.patch.object(session_file, 'OneViewClient')
+    def test_post_session(self, oneview_client_mockup):
+        """Tests post Session"""
+
+        # Loading Session mockup result
+        with open(
+            'oneview_redfish_toolkit/mockups_redfish/Session.json'
+        ) as f:
+            session_mockup = f.read()
+
+        # Create mock response
+        oneview_client = oneview_client_mockup()
+        oneview_client.connection.get_session_id.return_value = "sessionId"
+
+        # POST Session
+        response = self.app.post("/redfish/v1/SessionService/Sessions",
+                                 data=json.dumps(dict(
+                                     UserName="administrator",
+                                     Password="password")),
+                                 content_type='application/json')
+
+        # Gets json from response
+        json_str = response.data.decode("utf-8")
+
+        # Tests response
+        self.assertEqual(status.HTTP_200_OK, response.status_code)
+        self.assertEqual("application/json", response.mimetype)
+        self.assertEqual(session_mockup, json_str)
+        self.assertIn("/redfish/v1/SessionService/Sessions/1",
+                      response.headers["Location"])
+        self.assertEqual("sessionId", response.headers["X-Auth-Token"])
```

Comment:
sessionId is hardcoded to 1, how is it matching X-Auth-token and the test is succeeding??